docs/ibm-cloud-rules.md

```diff
@@ -1344,6 +1351,53 @@ components:
 </table>
 
 
+### Rule: operation-summary
+<table>
+<tr>
+<td><b>Rule id:</b></td>
+<td><b>operation-summary</b></td>
+</tr>
+<tr>
+<td valign=top><b>Description:</b></td>
+<td>This rule verifies that each operation has a non-empty.
```

Comment:
I think "summary" should be at the end of this sentence 🙂

Reply:
fixed